files/en-us/web/accessibility/aria/forms/basic_form_hints/index.html

```diff
@@ -104,7 +104,7 @@ <h2 id="Required_and_invalid_fields">Required and invalid fields</h2>
   var valid = emailValid(formData.email); // returns true if valid, false otherwise
 
   emailElement.setAttribute("aria-invalid", !valid);
-  setElementBorderColour(emailElement, valid); // sets the border to red if second arg is false
+  setElementBordercolor(emailElement, valid); // sets the border to red if second arg is false
```

Comment:
Not sure were this is defined originally, but I'm guessing it should stay camelCase if it can be renamed
```suggestion
  setElementBorderColor(emailElement, valid); // sets the border to red if second arg is false
```